php-practice/section7.php

```diff
@@ -76,3 +95,15 @@
  *
  * (It returns TRUE if $needle is a substring of $haystack; FALSE otherwise.)
  */
+
+function stringContains($haystack, $needle) {
+	$pos = strpos($haystack, $needle);
+
+		if ($pos == true) {
+			return true;
+		} else {
+			return false;
+		}
+}
+
+stringContains("This is the string", "ah");
```

Comment:
So you probably noticed that running `stringContains(...);` doesn't actually do anything on the console. In fact, it doesn't really do anything in the program. All `stringContains` is good for is returning true or false.

Here are a couple ways you could test your function to see if it works:

```PHP
$stringContainsOtherString = stringContains('This is the string', 'ah');

if ($stringContainsOtherString) {
  echo 'The substring was found';
} else {
  echo 'The substring was not found';
}
```

...or here's a more "shorthand" version of that using a ternary operator (The `?` `:` thing)...

```PHP
$stringContainsOtherString = stringContains('This is the string', 'ah');
echo $stringContainsOtherString ? 'The substring was found' : 'The substring was not found';
```

...or here's the super shorthand version of that:

```PHP
echo stringContains('This is the string', 'ah') ? 'The substring was found' : 'The substring was not found';
```